certified-connectors/Datablend/apiDefinition.swagger.json

```diff
@@ -0,0 +1,324 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Datablend",
+    "description": "Connect to the Datablend API for a cost-effective iPaaS solution supporting finance and accounting ",
+    "version": "1.0",
+	"contact":{
+		"name":"Datablend Support",
```

Comment:
@bmp02050 Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Changed Indentation to 2 space tab

Reply:
@bmp02050 Can you check the Indentation (property alignment) at line number 27 and fix similar alignment issues of all the properties in the swagger.

Reply:
Updated